Rollbar/RollbarConfig.cs

```diff
@@ -0,0 +1,31 @@
+namespace RollbarDotNet
+{
+    public class RollbarConfig
+    {
+        public RollbarConfig() : this("")
+        {
+        }
+
+        public RollbarConfig(string accessToken)
+        {
+            AccessToken = accessToken;
```

Comment:
Because .NET allows object initializers, it's uncommon to have a lot of constructors, because you can initialize everything like so

```
new RollbarConfig
{
    AccessToken = "",
    Environment = "",
    EndPoint = ""
}
```
 I provided the one constructor overload, just for simplicity to get going quickly.